app/views/bills/index.html.erb

```diff
@@ -32,5 +32,21 @@
       </select>
     </div>
   </div>
+  <div class="row">
+    <div class="col-6">
```

Comment:
This positioning looks strange on my screen. Can the top five bills section be centered? Or maybe left align the search by bill number section so it matches the top five below?

Reply:
can you post a screenshot of what you mean?